locales/en/public.json

```diff
@@ -0,0 +1,12 @@
+{
+  "About": "About",
+  "Bugs": "Bugs",
+  "CRYOSTAT_TRADEMARK": "Copyright The Cryostat Authors, The Universal Permissive License (UPL), Version 1.0",
+  "File a Report": "File a Report",
+  "Homepage": "Homepage",
+  "Known Issues": "Known Issues",
+  "License": "License",
+  "Mailing List": "Mailing List",
+  "Open Source License": "Open Source License",
+  "version": "version"
```

Comment:
I don't suppose this `i18n` library has support for splitting these keys up by component in some way?

```json
{
  "About": {
    "version": "version"
  },
  "Recordings": {
    "age": "age"
  }
}
```

or something like that would be nice. If not, even this could work:

```json
{
  "About.ABOUT": "About",
  "About.BUGS": "Bugs",
  "About.FILE_A_REPORT": "File a Report",
  ...
  "Recordings.AGE": "age"
}
```

I think keying the messages `LIKE_THIS` just makes it more clear to the reader (developer, reviewer) that what they're looking at is a localization key and not the actual message that will be rendered.